velox/functions/sparksql/registration/RegisterJson.cpp

```diff
@@ -24,6 +25,8 @@ void registerJsonFunctions(const std::string& prefix) {
       {prefix + "get_json_object"});
   registerFunction<JsonObjectKeysFunction, Array<Varchar>, Varchar>(
       {prefix + "json_object_keys"});
+  registerFunction<JsonArrayLengthFunction, int32_t, Varchar>(
+      {prefix + "json_array_length"});
```

Comment:
To avoid duplicating code, we would recommend extracting the implementation to the 'velox/functions/lib/' and registering it with corresponding return types for Presto and Spark if the semantics are the same (corner cases may need to be verified).

Reply:
@rui-mo The input type is a bit different, one is `Json`, another is `Varchar`.

Reply:
@rui-mo removed duplicated code. Please take a look again